contracts/bonding/BondingManager.sol

```diff
@@ -95,8 +82,7 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
 
     function BondingManager(address _controller) Manager(_controller) {}
 
-    function initialize(uint64 _unbondingPeriod, uint256 _numTranscoders, uint256 _numActiveTranscoders) external beforeInitialization returns (bool) {
-        finishInitialization();
```

Comment:
If we get rid of the `finishedINitialization()` call and the modifier that checks if the manager was initialized, are we still certain that the contract has been initialized and is in the proper state to be used?

Reply:
No, but the idea is that at genesis, since the Controller starts off as paused, all contract functions are disabled and the Controller's owner can "initialize" the contracts by calling `setParameters` on each of the contracts and then afterwords once the Controller is unpaused the contract will have been initialized with the correct parameter values. When we upgrade a contract, the proxy contract will still have the values set by that call to `setParameters`.

This safety here is very dependent on the owner of the Controller being the only address that is allowed to call this function, but I think we have that dependency no matter how we implement parameter updates